worlds/civ_6/Options.py

```diff
@@ -0,0 +1,105 @@
+from dataclasses import dataclass
+from Options import Choice, DeathLink, DefaultOnToggle, PerGameCommonOptions, Range, StartInventory, StartInventoryPool, Toggle
+
+
+class ProgressionStyle(Choice):
+    """Determines what progressive items (if any) should be included.\n\n
+    Districts Only: Each tech/civic that would normally unlock a district or district building now has a logical progression. Example: TECH_BRONZE_WORKING is now PROGRESSIVE_ENCAMPMENT\n\n
```

Comment:
I looked at how this option description looks in a template yaml, and the `\n\n` kinda looks like ass tbh
It looks like:
```yaml
    # Determines what progressive items (if any) should be included.
    # 

    # Districts Only: Each tech/civic that would normally unlock a district or district building now has a logical progression. Example: TECH_BRONZE_WORKING is now PROGRESSIVE_ENCAMPMENT
    # 

    # Eras and Districts: Players will be defeated if they play until the world era advances beyond the currently unlocked maximum era.
```

Reply:
Ah good call